types/webpack-dev-server/webpack-dev-server-tests.ts

```diff
@@ -212,6 +212,20 @@ const c6: WebpackDevServer.Configuration = {
         verbose: true,
     },
 };
+const c7: WebpackDevServer.Configuration = {
```

Comment:
Maybe instead of creating new test configs, you put `static` options in `c3` and `c4`  accordingly?